eq-author/src/App/questionPage/Routing/RoutingPage/RoutingEditor/RuleEditor/BinaryExpressionEditor/MultipleChoiceAnswerOptionsSelector.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import styled from "styled-components";
+
+import ToggleChip from "components/buttons/ToggleChip";
+import { includes, get } from "lodash";
+
+import { PropTypes } from "prop-types";
+
+const MultipleChoiceAnswerOptions = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  padding: 1em 0;
+`;
+
+class MultipleChoiceAnswerOptionsSelector extends React.Component {
+  static propTypes = {
+    expression: PropTypes.shape({
+      left: PropTypes.shape({
+        options: PropTypes.arrayOf(
+          PropTypes.shape({
+            id: PropTypes.string.isRequired,
+            label: PropTypes.string,
+          })
+        ),
+      }).isRequired,
+      right: PropTypes.shape({
+        options: PropTypes.arrayOf(
+          PropTypes.shape({
+            id: PropTypes.string.isRequired,
+          })
+        ),
+      }),
+    }).isRequired,
+    onRightChange: PropTypes.func.isRequired,
+  };
+
+  get selectedOptionIds() {
+    return get(this.props.expression.right, "options", []).map(({ id }) => id);
```

Comment:
```suggestion
    return get(this.props, "expression.right.options", []).map(({ id }) => id);
```

Reply:
The right is nullable but expression is not. So, I preferred only using get on the nullable part but am happy to change if you feel its still better.